scripts/mount-admin-tools-bucket.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# MIT License
+#
+# (C) Copyright [2024] Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+# Mount admin-tools S3 bucket
+s3_bucket="admin-tools"
+s3fs_mount_dir="/var/lib/admin-tools"
+s3_user="admin-tools"
+
+s3fs_cache_dir=/var/lib/s3fs_cache
```

Comment:
Allow overrides, could save someone the need to edit the script.

```suggestion
s3_bucket="${S3_BUCKET:-admin-tools}"
s3fs_mount_dir="${S3FS_MOUNT_DIR:-/var/lib/admin-tools}"
s3_user="${S3_USER:-admin-tools}"
s3fs_cache_dir="${S3FS_CACHE_DIR:-/var/lib/s3fs_cache}"
```
Also removes the new line separating these four variables just for the sake of bundling the assignments together.

Reply:
This suggestion is compatible with `set -u`.